src/applications/personalization/profile-2/components/connected-apps/ConnectedApps.jsx

```diff
@@ -151,11 +150,10 @@ export class ConnectedApps extends Component {
                   'profile-section': 'vets-faqs',
                 })
               }
-              href="/sign-in-faq/"
+              href="/sign-in-faq/#connecting-third-party-(non-VA"
```

Comment:
I know, this link looks super weird. 😄  But, I confirmed with Danielle, and the `(` is supposed to be there. She said it is part of the h2 it will link to.

Reply:
omg 🤦